energy-growers/src/components/AccordionList/AccordionList.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+import React, { Fragment, useRef, useState } from "react";
+import "./AccordionList.scss";
+import { AccordionList } from "@/models/accordion";
+import Accordion from "./Accordion/Accordion";
+
+const AccordionList: React.FC<AccordionList> = ({ items, title = "FAQ" }) => {
+  const [active, setActive] = useState<number>(-1);
```

Comment:
```suggestion
const refs = useRef<(HTMLDivElement | null)[]>([]);
  const handleRefAssignment =
    (index: number) => (ref: HTMLDivElement | null) => {
      refs.current[index] = ref;
    };
  const { isComponentVisible, setComponentVisible } = useComponentVisibles(
    false,
    refs
  );
```

gli hook vanno utilizzati, perche gia testati e funzionanti, non rinventiamoci la ruota